packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -605,6 +606,25 @@ class AccessibilityActionsExample extends React.Component {
             </View>
           </TouchableWithoutFeedback>
         </RNTesterBlock>
+
+        <RNTesterBlock title="Fake slider example">
+          <View
+            accessible={true}
+            accessibilityRole="adjustable"
+            accessibilityActions={[{name: 'increment'}, {name: 'decrement'}]}
+            onAccessibilityAction={event => {
+              switch (event.nativeEvent.actionName) {
+                case 'increment':
+                  Alert.alert('Alert', 'increment action success');
+                  break;
+                case 'decrement':
+                  Alert.alert('Alert', 'decrement action success');
+                  break;
+              }
+            }}>
+            <Text>Slider</Text>
+          </View>
+        </RNTesterBlock>
```

Comment:
This is my first time trying to add a test/example, woud I need to set the ` accessibilityState={{disabled: true}}` as  initially ? and how would i set it later on like when the slider is used should I alert that the state has changed

Reply:
I think the original issue is that the Slider component does not announce its selection on focus. So you'd want to render a Slider component here .. I'm not sure if there's a callback for an onFocus here. But at least with Talkback/VoiceOver on we'd want to confirm that the Slider state is being announced. Not sure the best way to illustrate that in the example

Reply:
Oh maybe I can ask @fabriziobertoglio1987 to assist us here

Reply:
@sladyn98 Here are the docs for the Slider component :pray: https://reactnative.dev/docs/next/slider

Reply:
I added an example here, how do I get i set the accessibility State here ? like it does not have a property pertaining to that

Reply:
Whoops sorry about the delay, forgot to hit submit!